### PR TITLE
metrics: stop kata components before start a metric test.

### PR DESCRIPTION
**Pull request by dborquez**
This PR kills all kata components before start a new metric test.

### CODE REVIEW

File: tests/common.bash [hunk 1]
```diff
@@ -186,31 +186,27 @@ function clean_env_ctr()
 	if (( count_tasks > 0 )); then
 		die "Can't remove running containers."
 	fi
-
-	kill_kata_components
```
Comment (fidencio) [inline tests/common.bash:190]: I'd say you could still leave it here.

File: tests/metrics/lib/common.bash [hunk 2]
```diff
@@ -192,6 +192,8 @@ function kill_processes_before_start()
 	CTR_PROCS=$(sudo "${CTR_EXE}" t list -q)
 	[[ -n "${CTR_PROCS}" ]] && clean_env_ctr
 
+	kill_kata_components
```
Comment (fidencio) [inline tests/metrics/lib/common.bash:195]: And also add it here. :-)

**Review by fidencio — APPROVED** [review]
lgtm, my comment is a nit and doesn't have to be addressed before merging this one.

File: tests/common.bash (hunk 1 above)
Comment (dborquez) [reply tests/common.bash:190]: The problem is that the purpose of `clean_env_ctr` is only to kill running containers, that's why we moved `kill_kata_components` to the stage where we need to stop kata running processes.

**Comment by GabyCT** [comment]
@dborquez seems like it needs a daemon-reload because we are getting the following failure
```
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
```

**Pull request merged**